lms/djangoapps/hera/templates/hera/course-lessons-outline-fragment.html

```diff
@@ -14,36 +14,40 @@
 subsection_new = subsection_vertical.get('children')
 %>
 <main role="main" class="course-outline" id="main" tabindex="-1">
-    % if course_sections is not None:
-        <button class="btn btn-primary"
-                id="expand-collapse-outline-all-button"
-                aria-expanded="false"
-                aria-controls="course-outline-block-tree"
-                >
-          <span class="expand-collapse-outline-all-extra-padding" id="expand-collapse-outline-all-span">${_("Expand All")}</span>
-        </button>
-        <ol class="block-tree accordion"
-            id="course-outline-block-tree"
-            aria-labelledby="expand-collapse-outline-all-button">
-
-        % for vertical in subsection_new:
-                                    <li class="vertical outline-item focusable">
-                                        <a class="outline-item focusable"
-                                            href="${ vertical['lms_web_url'] }"
-                                            id="${ vertical['id'] }">
-                                            <div class="vertical-details">
-                                              <div class="vertical-title">
-                                                ${ vertical['display_name'] }
-                                              </div>
-                                            </div>
-                        % if vertical.get('complete'):
-                                                <span class="complete-checkmark fa fa-check"></span>
-                        % endif
-                                        </a>
-                                    </li>
-                    % endfor
-        </ol>
-    % endif
+
+% if course_sections is not None:
+    <!-- select page -->
+    <div class="onboarding-holder">
```

Comment:
@madeira 
is class naming ok for Selection Page?

Reply:
For this PR it's ok - in further changes I will rename it for more common name